.github/workflows/build.yaml

```diff
@@ -46,10 +64,55 @@ jobs:
         PYTHON: python
     - name: Install dependencies
       run: make setup
+    - name: Create changelog and bump
+      if: ${{ github.ref == 'refs/heads/main' && matrix.os == env.ARTIFACT_OS && matrix.python == env.ARTIFACT_PYTHON }}
+      run: |
+        git config --global user.name "github-actions"
+        git config --global user.email "github-actions@github.com"
+        git config --list --global
+        # First generate the CHANGELOG diff to use as release notes.
+        mkdir -p ${{ env.ARTIFACT_DIR }}
+        cz changelog --dry-run --incremental --unreleased-version "Release Notes" > ${{ env.ARTIFACT_DIR }}/CHANGELOG.md
+        # Update the CHANGELOG.md.
+        cz changelog
+        cz bump
     - name: Build the package
-      # We don't need to check and test the package separately
-      # because `make dist` runs those targets first and only
-      # builds the package if they succeed.
-      run: make dist
-    - name: Build the docs
-      run: make docs
+      # We don't need to check and test the package separately because `make dist` runs
+      # those targets first and only builds the package if they succeed.
+      run: |
+        make dist requirements sbom
```

Comment:
I think this line makes issue https://github.com/jenstroeger/python-package-template/issues/261 obsolete.